modules/trusted-firmware-m/Kconfig.tfm

```diff
@@ -139,6 +139,21 @@ config TFM_ISOLATION_LEVEL
 	  As isolation levels 2 and 3 require PSA_API (TFM_IPC) support,
 	  force level 1 when TFM_IPC is not enabled.
 
+config TFM_ITS_NUM_ASSETS
+	int "Maximum number of assets to be stored in Internal Trusted Storage"
+	default 0
+	help
+	  When set to a non-zero value, overrides the platform's default maximum
+	  number of assets that can be stored in the Internal Trusted Storage (ITS).
+
```

Comment:
Sorry.

It will be confusing for a user to read his config file and see

TFM_ITS_NUM_ASSETS=0
TFM_ITS_MAX_ASSET_SIZE=0

Code, whenever possible, should be readable without having to read the docs.

We need to either modify the default to match the TF-M default (if that is sane, stable, and exists).
Or have a bool option that indicates whether or not the assets should be overridden.

TFM_ITS_NUM_ASSETS will depend on this option and be hidden from the config file when false.

Reply:
Ok, added two more variables.